src/shaders/disk_andrews2009_frag.glsl

```diff
@@ -47,14 +47,11 @@ void main(void)
 {
     // Compute the radius and height of this fragment
     float radius = sqrt(ModelPosition.x * ModelPosition.x + ModelPosition.y * ModelPosition.y);
-    float height = abs(ModelPosition.z);
+    float height = atan(abs(ModelPosition.z),radius);
```

Comment:
Humm. The use of atan2 in the shader will result in a very large performance loss. Lets rewrite Andrew's equation to use `z` instead (he has a lot of odd conventions if I recall correctly). We can just make note of this in the help text and on the SIMTOI wiki.